FSW/src/util/hkUtil.cpp

```diff
@@ -0,0 +1,184 @@
+/* hkUtil.cpp handles NS2 payload housekeeping
+ * Usage:
+ *  module functionality
+ *      system state data collection
+ *      temperature control
+ * 
+ * Modules encompassed:
+ *  houskeeping
+ *  thermal control
+ *
+ * Additional files needed for compilation:
+ *  none
+ */
+
+/* - - - - - - Includes - - - - - - */
+// C++ libraries
+
+// Other libraries
+
+// NS2 headers
+#include "../headers/hkUtil.hpp"
+
+
+/* Module Variable Definitions */
+static HousekeepingData hkData[HK_SAMPLES_TO_KEEP] = {}; // array to store housekeeping data
```

Comment:
`housekeepingData` for camelCase consistency.

Reply:
changed.